firmware/shared/src/io/io_can.h

```diff
@@ -57,8 +70,10 @@ void io_can_transmitMsgFromQueue(void);
  */
 void io_can_popRxMsgFromQueue(CanMsg *msg);
 
+#ifdef TARGET_EMBEDDED
 /**
  * Callback fired by config-specific interrupts to receive a message from a given FIFO.
  * @param msg CAN msg to be populated by RXed msg.
  */
 void io_can_msgReceivedCallback(CanMsg *rx_msg);
+#endif
```

Comment:
nit: missing newline (see other files as well)
```suggestion
#endif

```